test/database/src/main/kotlin/no/nav/amt/tiltak/test/database/data/TestData.kt

```diff
@@ -0,0 +1,183 @@
+	)
+
+	val ARRANGOR_2 = InsertArrangorCommand(
+		id = UUID.fromString("bd9ea032-03a3-4dbf-96ec-8a2a69b5c92a"),
+		overordnet_enhet_organisasjonsnummer = "922222222",
+		overordnet_enhet_navn = "Org Tiltaksarrangør 2",
+		organisasjonsnummer = "222222222",
+		navn = "Tiltaksarrangør 2"
+	)
+
+
+	val ARRANGOR_ANSATT_1 = InsertArrangorAnsattCommand(
+		id = UUID.fromString("6321c7dc-6cfb-47b0-b566-32979be5041f"),
+		personlig_ident = "123456789",
+		fornavn = "Ansatt 1 fornavn",
+		etternavn = "Ansatt 1 etternavn"
+	)
+
+	val ARRANGOR_ANSATT_1_ROLLE_1 = InsertArrangorAnsattRolleCommand(
+		id = UUID.fromString("e0a0eda3-9f72-4330-8e34-7fb5176fe123"),
+		arrangor_id = ARRANGOR_1.id,
+		ansatt_id = ARRANGOR_ANSATT_1.id,
+		rolle = "KOORDINATOR"
+	)
+
+	val ARRANGOR_ANSATT_1_ROLLE_2 = InsertArrangorAnsattRolleCommand(
+		id = UUID.fromString("9c7db3c9-bb61-460b-be2a-0fdbcae912ab"),
+		arrangor_id = ARRANGOR_2.id,
+		ansatt_id = ARRANGOR_ANSATT_1.id,
+		rolle = "VEILEDER"
+	)
+
+
+	val ARRANGOR_ANSATT_2 = InsertArrangorAnsattCommand(
+		id = UUID.fromString("a24e659c-2651-4fbb-baad-01cacb2412f0"),
+		personlig_ident = "326749823",
+		fornavn = "Ansatt 2 fornavn",
+		etternavn = "Ansatt 2 etternavn"
+	)
+
+	val ARRANGOR_ANSATT_2_ROLLE_1 = InsertArrangorAnsattRolleCommand(
+		id = UUID.fromString("9892eb04-19b8-4244-8953-90d75d20a50c"),
+		arrangor_id = ARRANGOR_1.id,
+		ansatt_id = ARRANGOR_ANSATT_2.id,
+		rolle = "VEILEDER"
+	)
+
+
+	val TILTAK_1 = InsertTiltakCommand(
+		id = UUID.fromString("b18fd670-3bc1-11ec-8d3d-0242ac130003"),
+		navn = "Tiltak1",
+		type = "AMO"
+	)
+
+	val GJENNOMFORING_1 = InsertGjennomforingCommand(
+		id = UUID.fromString("b3420940-5479-48c8-b2fa-3751c7a33aa2"),
+		tiltak_id = TILTAK_1.id,
+		arrangor_id = ARRANGOR_1.id,
+		navn = "Tiltaksgjennomforing1",
+		status = "GJENNOMFORES",
+		start_dato = LocalDate.of(2022, 2, 1),
+		slutt_dato = LocalDate.of(2050, 12, 30),
+		registrert_dato = LocalDate.of(2022, 1, 1),
+		fremmote_dato = LocalDate.of(2022, 2, 1)
+	)
+
+	val NAV_ANSATT_1 = InsertNavAnsattCommand(
+		id = UUID.fromString("91e1c2e6-83cf-46f9-b8e0-4a9192c7dcbe"),
+		nav_ident = "Z4321",
+		navn = "Vashnir Veiledersen",
+		telefonnummer = "88776655",
+		epost = "vashnir.veiledersen@nav.no"
+	)
+
+	val NAV_ANSATT_2 = InsertNavAnsattCommand(
+		id = UUID.fromString("5e8790a9-7339-4ea2-ae75-54aac33f6c4d"),
+		nav_ident = "Z1234",
+		navn = "Ola Nordmann",
+		telefonnummer = "99887654",
+		epost = "ola.nordmann@nav.no"
+	)
+
+	val NAV_KONTOR_1 = InsertNavKontorCommand(
+		id = UUID.fromString("09405517-99c0-49e5-9eb3-31c61b9579cf"),
+		enhet_id = "1234",
+		navn = "NAV Testheim"
+	)
+
+
+	// Bruker 1
+
+	val BRUKER_1 = InsertBrukerCommand(
+		id = UUID.fromString("23b04c3a-a36c-451f-b9cf-30b6a6b586b8"),
+		fodselsnummer = "12345678910",
+		fornavn = "Bruker 1 fornavn",
+		etternavn = "Bruker 1 etternavn",
+		telefonnummer = "73404782",
+		epost = "bruker1@example.com",
+		ansvarlig_veileder_id = NAV_ANSATT_1.id,
```

Comment:
Det ble litt rart å sette veileder her. En test lengre ned er avhengig av at veileder allerede er satt

Jeg tenker at det beste hadde vært at disse brukerene er nøytrale, og ikke har satt ting som ikke er innlysende når man bruker testdataene og som kan påvirke resultatet som skal valideres. Det kan være fint å spesifisere alt dette i selve testen så det ikke skjer ting som man ikke tenker over. Kunne for eksempel hatt en LagMinimalBrukerMedDefaultVerdier så man bare må oppgi de parameterne som man skal asserte på og som er relevante for testen